<root>
    <TreeNodesModel>
        <Action ID="SwitchROS2Controllers">
            <description>
                <p>
                    Given a list of controllers to deactivate and controllers to activate, and controllers
                    to start and stop, calls the controller manager's switch_controllers service.
                </p>
            </description>
            <input_port name="start_controllers" default="">The controllers to start.</input_port>
            <input_port name="stop_controllers" default="">The controllers to stop.</input_port>
            <input_port name="deactivate_controllers" default="">The controllers to deactivate.</input_port>
            <input_port name="activate_controllers" default="">The controllers to activate.</input_port>
        </Action>
        <Action ID="TeleoperateNonServo">
            <description>
                <p>
                    Non-MoveIt Servo implementation of the Teleoperate behavior.
                </p>
            </description>
            <input_port name="controller_name" default="">The controllers to use.</input_port>
        </Action>
    </TreeNodesModel>
</root>
